skietbaan_fe/src/components/Documents.js

```diff
@@ -1,102 +1,71 @@
+    SendLOS(){
+        let token =getCookie("token");
+        fetch("http://api.skietbaan.retrotest.co.za/api/Documents/SendLOS/" + token)
     }
 
+   
     render() {
-        const postItems = (
-            <Table striped hover condensed>
-                <tbody>
-                    {this.state.array.map((post, index) => (
-                        <tr key={post.id} onClick={() => this.SearchMember(index)}>
-                            <td>
-                                <b>{post.username}</b>
-                                <p>{post.email}</p>
-                            </td>
-                        </tr>
-                    ))}
-                </tbody>
-            </Table>
-        );
         return (
             <div className="documents-container">
-                <div className="page-name">
-                    <h2>Documents</h2>
-                </div>
-                <div className="documents-body">
-                    <div className="label-select-member">
-                        <Label>Select Member</Label>
-                    </div>
-                    <div className="search-member-name">
-                        <Input
-                            type="text"
-                            name="usernameValue"
-                            id="usernameValue"
-                            placeholder="Username or Email"
-                            value={this.state.usernamesValue}
-                            onChange={this.GetMembers} />
+                    <div className="docuements-heading">
+                        <div className="documents-text">Documents</div>
                     </div>
-                    <div className="table-search-members">
-                        {this.GetMembers}
-                        {postItems}
-                    </div>
-                    <br />
-                    <div>
-                        <div className="button-upload-document">
-                            <Button className="upload-document-button">UploadDocument</Button>
+                <div className="documents-body ">
+                        <div className="label-select-document">                        
+                                Documents have requirements                            
                         </div>
-                        <br />
-                    </div>
-                    <div className="button-upload-document">
-                        <Button className="send-email">SendEmail</Button>
-                    </div>
-                    <br />
+                                
+                        <div className="button-upload-document-3">
+                            <button className={this.state.value=="Document"? "btn-active send-email btn-bottom-3":"btn-default send-email btn-bottom-3"} onClick={this.state.value=="Document"? this.SendLOGS: null}> Letter of Good Standing {this.state.value=="Document"? <img src={require("../resources/sendDoc.png")}/>:null}</button>
```

Comment:
Split over multiple lines and images are supposed to be imported into variables 
Same for line 53 61 and 65